src/hevm/src/EVM.hs

```diff
@@ -1721,8 +1721,13 @@ notStatic continue = do
     then vmError StateChangeWhileStatic
     else continue
 
+burnSym :: SymWord -> EVM () -> EVM ()
+burnSym n continue = case maybeLitWord n of
+  Nothing -> continue -- smt query (TODO: no gas mode)
```

Comment:
Does this mean no gas is deduced when dealing with symbolic values?

Reply:
currently yes. But there should really be a query made here unless run in `--no-gas` mode.